sdk/communication/azure-communication-chat/README.md

```diff
@@ -75,9 +78,9 @@ String endpoint = "https://<RESOURCE_NAME>.communcationservices.azure.com";
 NettyAsyncHttpClientBuilder httpClientBuilder = new NettyAsyncHttpClientBuilder();
 HttpClient httpClient = httpClientBuilder.build();
 
-// Your user access token retrieved from your trusted service
-String token = "SECRET";
-CommunicationUserCredential credential = new CommunicationUserCredential(token);
+// Your access key retrieved from your trusted service
+String accessKey = "SECRET";
+CommunicationUserCredential credential = new CommunicationUserCredential(accessKey);
```

Comment:
`CommunicationUserCredential` actually takes a user access token, not the accessKey. This change here should be reverted, sorry about the confusion.

Reply:
Oh whoops that was my bad, will revert in next commit!